### PR TITLE
Add option to suppress the complaints from sub process.

### LINKED ISSUE #638: Turn off the error complaint when data reader is interrupted
If the data reader is interrupted  by ```break``` in the ```for``` loop or ```ctrl+c``` from keyboard when loading batch data iteratively, there would output a lot of error complaints which look like

```
Process Process-9:
Process Process-10:
Traceback (most recent call last):
Process Process-5:
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
Traceback (most recent call last):
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self._target(*self._args, **self._kwargs)
  File "/paddle_work/models/fluid/DeepASR/data_utils/data_reader.py", line 264, in ordered_processing_task
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/paddle_work/models/fluid/DeepASR/data_utils/data_reader.py", line 317, in batch_assembling_task
    for sample in sample_generator():
  File "/paddle_work/models/fluid/DeepASR/data_utils/data_reader.py", line 291, in _sample_generator
    sample = sample_queue.get()
  File "<string>", line 2, in get
  File "/usr/lib/python2.7/multiprocessing/managers.py", line 759, in _callmethod
    kind, result = conn.recv()
IOError: [Errno 104] Connection reset by peer

    while order_id != out_order[0]:
  File "<string>", line 2, in __getitem__
  File "/usr/lib/python2.7/multiprocessing/managers.py", line 759, in _callmethod
    self.run()
    self.run()
```

### PR DESCRIPTION
**Pull request by pkuyym**
Fixes #638

### CODE REVIEW

File: fluid/DeepASR/data_utils/data_reader.py [hunk 1]
```diff
@@ -133,6 +134,9 @@ class DataReader(object):
         shuffle_block_num (int): Block number indicating the minimum unit to do 
                                  shuffle.
         random_seed (int): Random seed.
+        verbose (int): Whether to suppress the complaints from sub-process. If
```
Comment (kuke) [inline fluid/DeepASR/data_utils/data_reader.py]: Why don't use ```bool```?

**Review by kuke — COMMENTED** [review]
Almost LGTM

File: fluid/DeepASR/data_utils/data_reader.py (hunk 1 above)
Comment (pkuyym) [reply fluid/DeepASR/data_utils/data_reader.py]: We may add more verbose level for debugging.

Comment (kuke) [reply fluid/DeepASR/data_utils/data_reader.py]: Then please modify the comments, don't use ```Whether to ...```

**Review by zhxfl — APPROVED** [review]

**Review by kuke — APPROVED** [review]

**Pull request merged**